queries/python/highlights.scm

```diff
@@ -324,14 +324,14 @@
           (expression_statement
             (assignment
               left: (identifier) @field))))
- (#lua-match? @field "^%u@!.*$"))
+ (#lua-match? @field "^%l.*$"))
 ((class_definition
   body: (block
           (expression_statement
             (assignment
               left: (_
                      (identifier) @field)))))
- (#lua-match? @field "^%u@!.*$"))
+ (#lua-match? @field "^%l.*$"))
```

Comment:
Is there any reason why these aren't using the same patterns as the attributes?

Reply:
I haven't touched Python in so long, isn't @ for decorators and other similar stuff? I don't see how it could appear here

Reply:
I'm okay with both, but is it normal for python to declare private fields right inside block and not through `__init__`?

Reply:
Idk, not my language :^)

Reply:
Neither do I :^)

hamsta will do the call then